### DISCUSSION TITLE
Toggle between different metrics in the main graph

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#117] [Anything Else]
"The main graph should be able to show both unique visitors and pageviews, depending on what the user selects."

**Update:** You can now choose a different metric to display on the top graph!

**Comment by kevquirk** [comment]
+1 for this.

**Comment by Cdingram** [comment]
+1 would really love to see page views instead of unique visitors

**Comment by inouire** [comment]
+1 , both metrics are veey relevant (and I think that they are the only one I use :)

**Comment by metmarkosaric** [comment]
Adding another vote for this: "are you planning displaying pageviews in the graph as well? That would be a nice feature, I'd love to see!"

**Reply by AllienWorks** [reply]
Just to clarify – seeing both unique and pageviews in the same graph :) Ideally by default (or maybe customizable in the settings)

**Reply by metmarkosaric** [reply]
Yeah, agree. We can include them both by default. No need for any options

**Comment by metmarkosaric** [comment]
Another vote for showing pageviews directly in the main graph: https://twitter.com/flowdee/status/1356099444467499013

**Comment by derWebdesigner** [comment]
+1 since I am almost only looking at pageviews because it gives me a better feeling of the actual usage

**Comment by orasik** [comment]
Would be great to select:
- Visits
- Views
- Bouncing rate

This will help to identify how the bouncing rate is improving over time.

**Comment by metmarkosaric** [comment]
"any chance you’ll ever show a graph for Conversion Rate? Would be amazing to actually see how it’s changing over time for each event. Quick mockup"

![E1hXCcmXEAYeq6R](https://user-images.githubusercontent.com/34340819/118404841-f488ce00-b674-11eb-87c6-c91101121561.jpg)

([request via twitter](https://twitter.com/linuz90/status/1393959788510392331))

**Reply by wpbft** [reply]
This one also with e-commerce revenue over time would be nice

**Reply by RobertJoonas** [reply]
One more request for CR on the graph today, via email

**Comment by MilllerTime** [comment]
Seeing a graph of page views over time would be excellent - I keep coming back to wanted this.

More generally, it would be great if I could select any of the metrics displayed above the graph (in any of the views) to switch the graph to display the selected metric.

**Reply by metmarkosaric** [reply]
thanks! this is on the way. see https://github.com/plausible/analytics/pull/1364

**Reply by MilllerTime** [reply]
Looks brilliant! Thank you :)

**Comment by metmarkosaric** [comment]
**Update:** You can now choose a different metric to display on the top graph!
